src/components/graph/GraphChunksVirtualizer.tsx

```diff
@@ -445,10 +447,37 @@ function TimeSelector(props: TimeSelectorProps) {
 		return Math.floor(from + (to - from) * (clickPoint / width));
 	}
 
+	function handleMouseDown(event: React.MouseEvent<HTMLDivElement>) {
+		if (
+			!rootRef.current ||
+			(event.target instanceof Node && !rootRef.current.contains(event.target))
+		)
+			return;
+
+		isDown.current = true;
+		document.addEventListener('mousemove', handleMouseDrag);
+		document.addEventListener('mouseup', handleMouseUp);
+	}
+
+	function handleMouseDrag(event: MouseEvent) {
+		if (!rootRef.current || !isDown.current) return;
+		event.preventDefault();
+		const pointerEl = pointerRef.current;
+		const clickedTime = getTimeOffset(event.pageX - (pointerEl?.offsetWidth || 6) / 2);
+		onClick(clickedTime);
```

Comment:
that's not really a dragging. it will lag and cause a lot of unnecessary requests.
we should use dragging from framer motion here